src/App.js

```diff
@@ -1,13 +1,28 @@
+import { Switch, Route } from "react-router-dom";
+import NavBar from "./Components/NavBar";
+import Movies from "./Components/Movies";
+import People from "./Components/People";
+import Locations from "./Components/Locations";
 import "./App.css";
 
 function App() {
   return (
     <div className="app">
+      <div className="nav">
```

Comment:
I would suggest placing the div with the 'nav' class inside of the NavBar component. It makes it more modular and avoid issues should the component get moved